sdk/src/wallet/migration/migrate_0.rs

```diff
@@ -74,8 +74,10 @@ fn migrate_account(account: &mut serde_json::Value) -> Result<()> {
         }
     }
 
-    for (_key, foundry) in account["nativeTokenFoundries"].as_object_mut().unwrap() {
-        migrate_native_token(foundry);
+    if let Some(native_token_foundries) = account.get_mut("nativeTokenFoundries") {
```

Comment:
Oh it was added later, right?

Reply:
Yes

Reply:
why we don't have the same for `incomingTransactions` and `transactions`?

Reply:
You mean the `let Some`? Because they have always been there so no need to check if they exist or not, since they have to exist.